buildSrc/src/main/groovy/com/uber/okbuck/composer/android/TrasformDependencyWriterRuleComposer.groovy

```diff
@@ -33,18 +33,24 @@ final class TrasformDependencyWriterRuleComposer extends AndroidBuckRuleComposer
         String output = "\$OUT"
         List<String> cmds = [
                 "echo \"#!/bin/bash\" > ${output};",
-                "echo \"set -e\" >> ${output};",
+                "echo \"set -ex\" >> ${output};",
 
-                "echo \"export IN_JARS_DIR=\\\$1\" >> ${output};",
-                "echo \"export OUT_JARS_DIR=\\\$2\" >> ${output};",
-                "echo \"export ANDROID_BOOTCLASSPATH=\\\$3\" >> ${output};",
+                "echo \"java " +
 
-                configFile != null ? "echo \"export CONFIG_FILE=\$SRCS\" >> ${output};" : "",
-                transformClass != null ? "echo \"export TRANSFORM_CLASS=${transformClass}\" >> ${output};" : "",
+                        "-Din_jars_dir=\"\\\$1\" " +
```

Comment:
system properties are usually camel case. We should add `okbuck.` prefix to them to better namespace them and prevent conflicts